dotnet-desktop-guide/net/wpf/events/how-to-add-an-event-handler-using-code.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "How to add an event handler using code"
+description: Learn how to add an event handler in code-behind for an element in Windows Presentation Foundation (WPF).
+ms.date: "01/24/2021"
+dev_langs:
+  - "csharp"
+  - "vb"
+helpviewer_keywords:
+  - "event handlers [WPF], adding"
+  - "XAML [WPF], adding event handlers"
+---
+<!-- The acrolinx score was 97 on 01/24/2021-->
+
+# How to add an event handler using code (WPF .NET)
+
+You can assign an event handler to an element in Windows Presentation Foundation (WPF) using markup or code-behind. Although it's customary to assign an event handler in Extensible Application Markup Language (XAML), sometimes you might have to assign an event handler in code-behind. For instance, when:
+
+- You assign an event handler to an element after the markup page that contains the element loads.
+- You add an element and assign its event handler after the markup page that will contain the element loads.
+- You define the element tree for your application entirely in code.
+
+[!INCLUDE [desktop guide under construction](../../includes/desktop-guide-preview-note.md)]
+
+## Syntax for event handler assignment
+
+C# supports event handler assignment using:
+
+- The `+=` operator, which is also used in the common language runtime (CLR) event handling model.
+- The <xref:System.Windows.UIElement.AddHandler%2A> method.
+
+VB supports event handler assignment using:
+
+- The <xref:System.Windows.UIElement.AddHandler%2A> method, together with the [AddressOf](/dotnet/visual-basic/language-reference/operators/addressof-operator) operator to reference the event handler.
+- The [Handles](/dotnet/visual-basic/language-reference/statements/handles-clause) keyword in the event handler definition. For more information, see [Visual Basic and WPF event handling](/dotnet/desktop/wpf/advanced/visual-basic-and-wpf-event-handling?view=netframeworkdesktop-4.8&preserve-view=true).
+
+> [!NOTE]
+> In the following example, the C# code uses the `+=` operator, and the VB code uses the `AddHandler` method with `AddressOf` operator. Choose the language selector at the top of the page to select either C# or VB for the code sample.
+
```

Comment:
I don't really think this note is needed. Either way, they'll only see what is appropriate for their chosen language. I'm not sure what benefit it does to call it out.

```suggestion
```